shinydashboard/lantern/modules/fieldsmodule.r

```diff
@@ -0,0 +1,115 @@
+# Fields Module
+
+fieldsmodule_UI <- function(id) {
+  
+  ns <- NS(id)
+  
+  tagList(
+    h1("FHIR Capability Statement Fields"),
+    p("This is the list of fields included in the FHIR capability statements from the endpoints."),
+    # Hides the "exist" column which is necessary for grouping data but not necessary to display
+    tags$style(HTML("
```

Comment:
I'd recommend removing the `exist` column in the `get_capstat_fields_count` function, rather than using a CSS selector to hide the column. This is dependent on `exists` being the first column, and would break if we wanted something else as the first column.

Reply:
This is a good point, this change should now be addressed, let me know if i missed anything!